capstone/src/main/webapp/dashboard.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Dashboard</title>
+    <link rel="stylesheet" href="/style.css">
+    <!-- TODO(biancamacias): revist url to add api key -->
+    <!-- <script src="https://maps.googleapis.com/maps/api/js?key="></script> -->
```

Comment:
nit - avoid merging commented out code. i think it will make more sense just to add this resource in the PR that introduces the dependency.